packages/gatsby-remark-prismjs/src/load-prism-language-extension.js

```diff
@@ -0,0 +1,64 @@
+const Prism = require(`prismjs`)
+const loadPrismLanguage = require(`./load-prism-language`)
+const replaceStringWithRegex = require(`./replace-string-with-regexp`)
+
+module.exports = languageExtensions => {
+  //  We only accept array or object as the 'languageExtensions' config property
+  //  (Note that 'instanceof Object' will return true for both Array and Object)
+  if (!(languageExtensions instanceof Object)) {
+    console.log(
+      `'languageExtensions' needs to be defined as an array or a JSON object: `,
+      languageExtensions
+    )
+    throw new Error(
+      `'languageExtensions' needs to be defined as an array or a JSON object.`
+    )
+  }
+
+  //If an object is given in the config we convert it to an array with a single object.
+  if (!(languageExtensions instanceof Array)) {
+    languageExtensions = [languageExtensions]
+  }
+
+  languageExtensions.forEach(l => {
+    loadLanguageExtension(l)
+  })
+}
+
+let loadLanguageExtension = languageExtension => {
+  if (!containsMandatoryProperties(languageExtension)) {
+    console.log(
+      `A languageExtension needs to contain 'language' and 'extend' or both and a 'definition'. Given config will not be loaded: `,
+      languageExtension
```

Comment:
Is it valuable to log the whole object? What will this look like?

In other words--it may be more valuable to refer to the language or something to illustrate the config.